packages/lesswrong/components/search/SearchPageTabbed.tsx

```diff
@@ -62,7 +62,8 @@ const styles = (theme: ThemeType): JssStyles => ({
     },
   },
   filtersHeadline: {
-    marginBottom: 18
+    marginBottom: 18,
+    fontFamily: isEAForum ? theme.palette.fonts.sansSerifStack : undefined,
```

Comment:
I would make this bolder, it feels too thin now (maybe `fontWeight: 500`). FYI you also don't need to forum-gate things in this file, as we are currently the only site that uses this file.